frontend/public/components/utils/mask-data.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { CopyToClipboard } from './index';
+
+export const MaskedData = () => <React.Fragment>
+  <span className="sr-only">Value hidden</span>
+  <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
+</React.Fragment>;
```

Comment:
Since this is exactly the same as the one in secret data, we should be able to reuse this there instead of copying.

Reply:
I moved it from secretData to the new MaskData component, seems more appropriate in the new component.